docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1454,17 +1455,19 @@ must restart the server for that change to take effect.
 
 ### 5.2.2 WebApp security
-By default, the FHIR server REST API is available on both HTTP and HTTPS endpoints. In addition, the web application (WAR) that contains the FHIR server's REST API implementation is secured via client certificate-based authentication. As a backup to the client authentication scheme, basic authentication is also supported. If the client chooses to use OAuth 2.0 authentication using a Bearer Token mechanism, this can be done as well and is described in more detail in [Section 5.2.5 Oauth 2.0](#525-oauth-20).
+By default, the FHIR server REST API is only available via HTTPS on port 9443 and protect by HTTP basic authentication.
+Alternatively, the server can use OpenID Connect and OAuth 2.0 via a Bearer Token as described in [Section 5.2.4 Oauth 2.0](#524-oauth-20).
+In addition, the FHIR server web application can be secured via client certificate-based authentication.
 
 Here are some notes related to these authentication schemes:
-*   Basic authentication is a very simple authentication scheme and can be used on both HTTP or HTTPS endpoints, although it is extremely insecure when used with an HTTP endpoint since the user and password information is essentially transmitted in plain text.
-*   Client certificate-based authentication can only be used in conjunction with an HTTPS endpoint since it involves SSL handshake negotiations.
-*   The main value of client authentication is that the server is able to securely authenticate the client through the use of certificates.
-*   OAuth 2.0 authentication can only be used in conjunction with an HTTPS endpoint, because OAuth authorization steps rely on SSL handshake negotiations.
+*   Basic authentication is a very simple authentication scheme and should only be used over HTTPS because the username and password are essentially transmitted in plain text.
+*   OAuth 2.0 authentication can only be used in conjunction with an HTTPS endpoint because the OAuth authorization steps rely on SSL handshake negotiations.
+*   Client certificate-based authentication can only be used in conjunction with an HTTPS endpoint since it involves SSL handshake negotiations. The main value of client authentication is that the server is able to securely authenticate the client through the use of certificates.
 
-### 5.2.3 Server configuration steps
+### 5.2.3 Configuring mutual TLS authentication
 To properly configure the FHIR server's keystore and truststore files, perform the following steps:
```

Comment:
```suggestion
To properly configure the FHIR server's keystore and truststore files, perform the following steps.
```

end in a period, since the other parts are now no longer build points rather sections